From 39a9aec0fdca953035b0022a1f2c2d2c3e91c517 Mon Sep 17 00:00:00 2001
From: Donald McIntyre <donald@etherplan.com>
Date: Fri, 8 Nov 2019 16:47:06 -0700
Subject: [PATCH 1/3] Proposing Subeditor Position

I am proposing the "subeditor" position, a person, technical or not, who checks and corrects the text and format of ECIPs also writing additions or deletions, specifically non-code non-software ECIPs, or copies improvement proposals from other improvement proposal repos, to help off-load work from, and assist editors in correcting and approving such ECIPs.

I am also proposing myself as a subeditor as I have been helping editors and ECIP authors in this capacity, with their consent, this year. I think it would help to accelerate the input of ECIPs as editors often don't have time to go over the flow, and has proven useful up to now.
---
 _specs/ecip-1000.md | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/_specs/ecip-1000.md b/_specs/ecip-1000.md
index b5b50dd..07841bd 100644
--- a/_specs/ecip-1000.md
+++ b/_specs/ecip-1000.md
@@ -68,7 +68,7 @@ A good reason to transfer ownership is because the original author no longer has
 
 **If you are interested in assuming ownership of a ECIP**, send a message asking to take over, addressed to both the original author and the ECIP editor. If the original author doesn't respond to email in a timely manner, the ECIP editor will make a unilateral decision (it's not like such decisions can't be reversed :).
 
-## ECIP Editors
+## ECIP Editors & Subeditors
 
 The current ECIP editors are:
 
@@ -78,7 +78,11 @@ The current ECIP editors are:
 * Afri Schoedon (@soc1c)
 * Yaz Khoury (@YazzyYaz)
 
-## ECIP Editor Responsibilities & Workflow
+The current ECIP subeditors are:
+
+* Donald McIntyre (@TokenHash)
+
+## ECIP Editor & Subeditors Responsibilities & Workflow
 
 ### Introduction
 
@@ -102,6 +106,8 @@ ECIP editors are intended to fulfill administrative and editorial responsibiliti
 * Merge the pull request when it is ready.
 * List the ECIP in [[README.mediawiki]]
 
+ECIP subeditors are technical person or not, who check and correct the text and format of ECIPs also writing additions or deletions, specifically non-code non-software ECIPs, or copy improvement proposals from other improvement proposal repos, to help off-load work from, and assist editors in their duties and approving ECIPs.
+
 # ECIP Format and Structure
 
 ## Specification

From 20f273bc8d1c6f4c822d045250f8f46cea0ade52 Mon Sep 17 00:00:00 2001
From: Wei Tang <hi@that.world>
Date: Sun, 10 Nov 2019 21:49:59 +0900
Subject: [PATCH 2/3] Update ecip-1000.md

---
 _specs/ecip-1000.md | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/_specs/ecip-1000.md b/_specs/ecip-1000.md
index 07841bd..7839023 100644
--- a/_specs/ecip-1000.md
+++ b/_specs/ecip-1000.md
@@ -68,7 +68,7 @@ A good reason to transfer ownership is because the original author no longer has
 
 **If you are interested in assuming ownership of a ECIP**, send a message asking to take over, addressed to both the original author and the ECIP editor. If the original author doesn't respond to email in a timely manner, the ECIP editor will make a unilateral decision (it's not like such decisions can't be reversed :).
 
-## ECIP Editors & Subeditors
+## ECIP Editors & Coordinators
 
 The current ECIP editors are:
 
@@ -78,11 +78,11 @@ The current ECIP editors are:
 * Afri Schoedon (@soc1c)
 * Yaz Khoury (@YazzyYaz)
 
-The current ECIP subeditors are:
+The current ECIP coordinators are:
 
 * Donald McIntyre (@TokenHash)
 
-## ECIP Editor & Subeditors Responsibilities & Workflow
+## Responsibilities & Workflow
 
 ### Introduction
 
@@ -106,7 +106,7 @@ ECIP editors are intended to fulfill administrative and editorial responsibiliti
 * Merge the pull request when it is ready.
 * List the ECIP in [[README.mediawiki]]
 
-ECIP subeditors are technical person or not, who check and correct the text and format of ECIPs also writing additions or deletions, specifically non-code non-software ECIPs, or copy improvement proposals from other improvement proposal repos, to help off-load work from, and assist editors in their duties and approving ECIPs.
+ECIP coordinators are technical person or not, who check and correct the text and format of ECIPs also writing additions or deletions, specifically non-code non-software ECIPs, or copy improvement proposals from other improvement proposal repos, to help off-load work from, and assist editors in their duties and approving ECIPs.
 
 # ECIP Format and Structure
 

From 23cb247eaccaae86247b27d9557adad76582a59c Mon Sep 17 00:00:00 2001
From: Donald McIntyre <donald@etherplan.com>
Date: Mon, 11 Nov 2019 10:03:05 -0700
Subject: [PATCH 3/3] Dropping as a candidate

Dropping my name as a candidate, recommend to put 2 persons in that position if approved.
---
 _specs/ecip-1000.md | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/_specs/ecip-1000.md b/_specs/ecip-1000.md
index 7839023..65b7821 100644
--- a/_specs/ecip-1000.md
+++ b/_specs/ecip-1000.md
@@ -80,7 +80,7 @@ The current ECIP editors are:
 
 The current ECIP coordinators are:
 
-* Donald McIntyre (@TokenHash)
+* 
 
 ## Responsibilities & Workflow
 
